runtime/tests/integration_tests.rs

```diff
@@ -136,13 +136,10 @@ fn test_verify_cmd() {
         ],
     };
 
-    let resp = model
+    model
         .mailbox_execute(u32::from(CommandId::ECDSA384_VERIFY), cmd.as_bytes())
-        .unwrap()
         .unwrap();
-    let resp_bytes: &[u8] = resp.as_bytes();
```

Comment:
Should we assert that no response is provided? Also fine if we decide it's not worth the hassle.

Reply:
Done